### ISSUE TITLE
Incorrect deps edn example

### ISSUE THREAD

**Issue by zaibacu** [jonase/eastwood#322]
```
If you're using deps.edn, you can set options to eastwood linter in a EDN map, like this:

{:aliases
  {:eastwood
      {:main-opts ["-m" "eastwood.lint" {:source-paths ["src"]}]
	   :extra-deps {jonase/eastwood {:mvn/version "RELEASE"}}}}}

to your deps.edn, and you should then be able to run Eastwood as

$ clj -A:eastwood
```

Results in:

```
Exception in thread "main" clojure.lang.ArityException: Wrong number of args (2) passed to: eastwood.lint/-main
	at clojure.lang.AFn.throwArity(AFn.java:429)
	at clojure.lang.AFn.invoke(AFn.java:36)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.core$apply.invokeStatic(core.clj:665)
	at clojure.main$main_opt.invokeStatic(main.clj:491)
	at clojure.main$main_opt.invoke(main.clj:487)
	at clojure.main$main.invokeStatic(main.clj:598)
	at clojure.main$main.doInvoke(main.clj:561)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.main.main(main.java:37)
```

**Comment by jafingerhut** [comment]
@nmoya Have you tested with this change, and it works for you as described in the README?  If so, what operating system and other software versions are you using where you see it work?

I tried it on a system with this software, and it gave an error similar to the one shown in the comment above.

Ubuntu 18.04 Linux
OpenJDK 11.0.3
Clojure 1.10.1
clj "1.10.1.447"

The older instructions in the README do work for me, with the commas replacing spaces inside of a string argument.

**Comment by nmoya** [comment]
@jafingerhut @zaibacu Hey,

Yes, I tested and it works.

I think the issue is the Eastwood version. The official latest release is 0.3.5, which is before my merge. The repository owners need to publish a new version with the latest changes. Right now it is in an inconsistent state, the readme is (say v0.3.6), but the code being imported is still on v0.3.5.

To unblock you, you have two options:
1) Keep using the old readme, which is consistent with the code 0.3.5;
2) Use my commit (or latest), as the dependency version in your project;
For example:
```
{:extra-deps
   {jonase/eastwood {:git/url "https://github.com/jonase/eastwood"
                     :sha "4d1d05cc6de6ce565207d9a3c86864364f53f006"}}
   :jvm-opts ["-Xmx512m"]
   :main-opts ["-m" "eastwood.lint" {:source-paths ["src" "test" "dev"]
                                     :config-files ["resources/eastwood_config.clj"]
                                     :exclude-linters [:unused-ret-vals]}]}
```


If using option 2 still doesn't work, then I can share the versions of my other dependencies (OS, Clojure, Java, etc).

**Comment by slipset** [comment]
I've released a new version of eastwood, 0.3.6. LMK if that doesn't solve the problem
Sorry for the inconvenience.
